blockchain/account/accounts.go

```diff
@@ -61,6 +63,7 @@ type Manager struct {
 	acpIndexCap  uint64 // points to end of block
 }
 
+// IndexAccounts will set Manager.indexer
```

Comment:
FYI, it is more common to use the simple present tense in the comments, i.e., IndexAccounts sets Manager.indexer.

Reply:
fixed, plz check